ethereum/contracts/test/MangataToken.sol

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.6.2;
+
+import "@openzeppelin/contracts/presets/ERC20PresetFixedSupply.sol";
+
+contract MangataToken is ERC20PresetFixedSupply {
+
+		constructor(
```

Comment:
Here and later- please use spaces instead of tabulation. 
I'm aware we don't yet have the style guidelines set, but this one makes reviews difficult especially in .js files

Reply:
You want me to use spaces instead of tabulation in all code henceforth?
How many spaces do you want me to use as indentation?

Reply:
Yes, spaces is a way to go. (I believe Substrate has tabs by default, so we may have a discussion about it later)
But for solidity, js and ts- for sure. The exact number should be the standard for the language, see no reason to deviate from it. Thus, solidity- 4 spaces, js/ts 2 spaces (unless defined otherwise by Ondrej)

Reply:
Okay, I'll do that.